README.md

```diff
@@ -72,7 +72,12 @@ Usage: X-Ray [options]
 
 ### Windows  
 
-`make build-windows` would build binaries and .zip files in `/build` folder for the Windows platform.  
+`make build-windows` would build binaries and .zip files in `/build` folder for the Windows platform. 
+
+## Build with ARM
+GOARCH=arm64 would build daemon for arm architecture. Instruction to build daemon on linux OS for arm64 architecture, the build instruction would look something like this. Similarly, by changing GOARCH=arm64 should be able to build daemon on windows and macos too.
```

Comment:
You mean by changing the **`GOOS`** parameter value to `darwin` or `windows`, we can build for MacOS or Windows respectively as well right?